build/networking/README_NATIVE.md

```diff
@@ -21,53 +31,40 @@ root@linux:~# git clone https://github.com/ipdk-io/ipdk.git
 Without a proxy:
 
 ```bash
-root@linux:~# SCRIPT_DIR=<CLONE-PATH>/ipdk/build/networking/scripts <CLONE-PATH>/ipdk/build/networking/scripts/host_install.sh
+root@linux:~# <CLONE-PATH>/ipdk/build/networking/scripts/host_install.sh -d <CLONE-PATH>
```

Comment:
I think this is a typo for `SCRIPTS_DIR`, which is referenced in `host_install.sh`.